Anytype/Sources/PresentationLayer/Modules/Discussion/Container/DiscussionSpacingContainer.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import SwiftUI
+
+struct DiscussionSpacingContainer<Content: View>: View {
+    
+    let content: Content
+    @Environment(\.anytypeNavigationPanelSize) var navigationSize
+    
+    init(@ViewBuilder content: () -> Content) {
+        self.content = content()
+    }
+    
+    var body: some View {
+        GeometryReader { reader in
+            GeometryReader { readerIgnoreKeyboard in
+                content
+                    .safeAreaInset(edge: .bottom, spacing: 0) {
+                        Color.clear.frame(
+                            height: reader.safeAreaInsets.bottom == readerIgnoreKeyboard.safeAreaInsets.bottom
+                            ? reader.safeAreaInsets.bottom + navigationSize.height
+                            : reader.safeAreaInsets.bottom
```

Comment:
It will be nice to move this formula elsewhere and name it with descriptive name
using `let` constant of function 🙏

Reply:
It will be `func` with area arguments, not `let`. I will make 👍